ext/web/13_message_port.js

```diff
@@ -160,12 +184,19 @@ class MessagePort extends EventTarget {
           );
         } catch (err) {
           if (ObjectPrototypeIsPrototypeOf(InterruptedPrototype, err)) break;
+          if (this[_nodeWorker]) this.dispatchEvent(new Event("close"));
```

Comment:
Now this is tricky and I'm not yet sure how to handle that. Maybe we keep `this[_nodeWorker]` but make it non-enumerable and then we set it to true only from `node:worker_threads`

Reply:
I'm still thinking that we could handle this by adding `this[_workerThreadCloseCb]` that would be set from `node:worker_threads` and it will be up to the callback to decide what to do (eg. `this.emit("close")`).

Reply:
Sure. A callback should be fine too. I'll look into it.